docs/installation.rst

```diff
@@ -103,16 +103,16 @@ Debian/Ubuntu
 
 .. code-block:: console
 
-    $ sudo apt-get install build-essential libssl-dev libffi-dev python3-dev
+    $ sudo apt-get install build-essential libssl-dev libffi-dev \
+        python3-dev cargo
 
-RHEL/CentOS
-~~~~~~~~~~~
+Fedora/RHEL 8/CentOS 8
```

Comment:
```
/home/runner/work/cryptography/cryptography/docs/installation.rst:110: Spell check: RHEL: Fedora/RHEL 8/CentOS 8.
/home/runner/work/cryptography/cryptography/docs/installation.rst:110: Spell check: CentOS: Fedora/RHEL 8/CentOS 8.
```
Looks like you'll need to add `RHEL` and `CentOS` to `docs/spelling_wordlist.txt`